.clang-format

```diff
@@ -0,0 +1,2 @@
+BasedOnStyle: LLVM
```

Comment:
Hey, I left this on another note: the reason I didn't include this is because copybara from internal can't apply LLVM clang-tidy, so this will cause a mismatch on what's exported from internal.